### PR TITLE
add comments to audit module

### PR DESCRIPTION
**Pull request by infofromca**

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecord.cs [hunk 1]
```diff
@@ -2,21 +2,64 @@
...
 
 namespace Orchard.AuditTrail.Models {
+    /// <summary>
+    /// Audit Trail Event Record in the database.
+    /// </summary>
     public class AuditTrailEventRecord {
         public virtual int Id { get; set; }
+
+        /// <summary>
+        /// The happend time of the event.
```
Comment (DanielStolt) [inline src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecord.cs]: Bad English, rephrase as:
"The time when the event occurred."

File: src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecord.cs [hunk 2]
```diff
@@ -2,21 +2,64 @@
...
+        /// </summary>
         public virtual DateTime CreatedUtc { get; set; }
+
+        /// <summary>
+        /// The user name of the user who made the event..
```
Comment (DanielStolt) [inline src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecord.cs]: Better phrased as:
"The user name of the user whose change caused the event to occur."
And remove double periods at the end.

File: src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecord.cs [hunk 3]
```diff
@@ -2,21 +2,64 @@
...
 
+        /// <summary>
+        /// The comment of the event.
+        /// </summary>
+
         [StringLengthMax]
         public virtual string Comment { get; set; }
+
+        /// <summary>
+        /// The ip address of the user who made the event.
```
Comment (DanielStolt) [inline src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecord.cs]: Rephrase as:
"The IP adress of the user who caused the event to occur."

File: src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecordResult.cs [hunk 4]
```diff
@@ -1,6 +1,16 @@
 namespace Orchard.AuditTrail.Models {
```
Comment (DanielStolt) [inline src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecordResult.cs]: Apply trailing periods consistently.

File: src/Orchard.Web/Modules/Orchard.AuditTrail/Services/IEventDataSerializer.cs [hunk 5]
```diff
@@ -1,8 +1,22 @@
 using System.Collections.Generic;
 
 namespace Orchard.AuditTrail.Services {
+    /// <summary>
+    /// Serialize or deserialize event data.
```
Comment (DanielStolt) [inline src/Orchard.Web/Modules/Orchard.AuditTrail/Services/IEventDataSerializer.cs]: Dependencies should be described with more context IMO. Perhaps better phrased as something like:
"A service responsible for serializing and deserializing audit trail event data."

**Comment by DanielStolt** [comment]
I like the initiative of providing XML comments!

I've written a few comments - if you could go over the PR and make it a bit more consistent with regards to punctuation etc, that would be truly great!

File: src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecordResult.cs (hunk 4 above)
Comment (infofromca) [inline src/Orchard.Web/Modules/Orchard.AuditTrail/Models/AuditTrailEventRecordResult.cs]: applied

**Comment by DanielStolt** [comment]
Nice work, thanks.

**Pull request merged**